src/RelationalModel/Relation.js

```diff
@@ -100,14 +101,25 @@ var Relation = module.exports = kind(
 	/**
 	* @private
 	*/
+	isRelated: function (related) {
+		return related === this.related;
+	},
+
+	/**
+	* @private
+	*/
 	getRelated: function () {
 		return this.related;
 	},
 
 	/**
+	* Sets the `related` model
+	*
+	* @param {[type]} related - The related model
+	* @param {[type]} [opts] - Used by subkinds to configure the behavior of the method
```

Comment:
I think related should probably be {module:enyo/Model~Model}  and opts should be {Object} or, if we have a typedef form options, that.